keyboards/crkbd/keymaps/rarick/flash.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+if [ $# -ne 3 ]; then
+  echo "Usage: source flash.sh BUS DEVICE HEX_FILE"
+fi
+
+sudo dfu-programmer "atmega32u4:$1,$2" erase --force
+sudo dfu-programmer "atmega32u4:$1,$2" flash --debug 3 --eeprom "$3"
+sudo dfu-programmer "atmega32u4:$1,$2" flash --debug 3 "$3"
```

Comment:
This isn't needed, you can just run `qmk flash -kb crkbd/rev1 -km rarick`.

Reply:
I can remove the script from the PR, but I needed this to flash on my machine.

I'm running Arch, and despite changing the `udev` rules to allow read/write to the device by my user (and I tested with `777` permissions as wel), it didn't have any effect. Didn't want to install the QMK flasher onto root, so I just used this script to flash the devices.

Reply:
`flash.sh` has been removed :heavy_check_mark:

Reply:
`qmk flash` with `BOOTLOADER = atmel-dfu` pretty much just runs these commands anyway, so I don't see how that is possible.